bindings_ffi/src/mls.rs

```diff
@@ -70,6 +70,7 @@ pub async fn create_client(
     account_address: String,
     nonce: u64,
     legacy_signed_private_key_proto: Option<Vec<u8>>,
+    history_sync_url: Option<String>,
```

Comment:
Where does this URL come from?

Reply:
Integrators will supply it when they create a client-app - if they want to run their own Msg History Server; else, when we stand up our own, we can point to ours as a fallback/default.